packages/to-dts/lib/cli.js

```diff
@@ -23,27 +23,34 @@ const toDts = {
         describe: 'Type of export',
         type: 'string',
       },
-      output: {
+      'output.file': {
         alias: 'o',
         describe: 'File to write to',
         type: 'string',
       },
       includeDisclaimer: {
         describe: "Include disclaimer in output file that it's automatically generated",
         type: 'boolean',
-        default: true,
+        default: undefined,
+      },
+      config: {
+        alias: 'c',
+        describe: 'Path to config file',
+        type: 'string',
+        default: null,
       },
```

Comment:
consider if using the builtin config support in yargs is a good options, (using y.config(...)) (see nebula.js for example)

Reply:
Good idea. I have updated the toDts module to use `yargs.config(...)`.

There looks to be some slight differences between nebula.js and scriptappy. For example, default values are kept in the config file instead as a `default` property on the options. In the `handler()` function the default config is merged with the arguments (including ones read from the config file):

```js
const config = extend(true, {}, defaultConfig, argv);
```